### DISCUSSION TITLE
User cannot find my server on Classical/Folk tab

### DISCUSSION THREAD

**Discussion by johnburt** [jamulussoftware/jamulus#3449] [Technical Support & Help] [answered]
A new user cannot find my server on its usual tab (usually the Classical/Folk tab). If I move my server to another tab, say to AnyGenre 1,  he still can't find it. But other servers appear on both tabs for him, and we can connect to those other servers. Other users have been able to connect to my server regularly. And I have no problem finding the server or connecting to it from another computer at my home. When the user goes to https://explorer.jamulus.io/ my server shows up there, and he can raise the welcome message from it. I can't for the life of me figure out what the issue is. But I am planning a session with many others organized by this user, and it is essential that he be able to connect. 

If he restarts his client from the command line with ---showallservers my server shows up, but it also shows no ping time, and he can't connect to it.

**Comment by dcorson-ticino-com** [comment, marked as answer]
This sounds strangely like a network problem between your server and the user.
If your server can be seen over the explorer running on the users compi it shows that the server is working, but the network connection is passing through the explorers server, not directly.
Some providers do not treat UDP packets, which Jamulus uses, very well which causes problems.
There is little that one can do except changing internet provider.  To see if that is really the problem the user can try using a hotspot through his mobile phone to connect.  That will not give a low latency connection, but will go through a different provider and if they can connect, even with poor latency, it is pretty much proven that the provider is the problem.

**Reply by johnburt** [reply]
Sure enough, when my friend connects through his phone he can indeed see my server.
